README.md

```diff
@@ -1,4 +1,4 @@
-# [React Color](http://casesandberg.github.io/react-color/)
+# [React Spectrum](http://socialtables.github.io/react-spectrum/)
```

Comment:
I would add a sentence mentioning that it was originally forked from react-color (just to give some credit to the original creator)